libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -366,18 +362,18 @@ float SoaringController::McCready(float alt)
     return thermal_vspeed;
 }
 
-SoaringController::ActiveStatus SoaringController::active_state() const
+SoaringController::ActiveStatus SoaringController::active_state(bool override_disable) const
```

Comment:
```suggestion
SoaringController::ActiveStatus SoaringController::active_state(const bool force_disable) const
```
"override_disable" is a bit vague. Could be interpreted as we're overriding the ability to disable.